src/components/TodoList/TodoList.jsx

```diff
@@ -1,44 +1,110 @@
 import React from 'react';
 import './TodoList.scss';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
 
-export const TodoList = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
+export class TodoList extends React.Component {
+  state = {
+    query: '',
+    status: 'all',
+  };
 
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
+  handleChange = (event, field) => {
+    this.setState({
+      [field]: event.target.value,
+    });
+  };
 
-          <button
-            className="
-              TodoList__user-button
-              TodoList__user-button--selected
-              button
-            "
-            type="button"
-          >
-            User&nbsp;#1
-          </button>
-        </li>
+  render() {
+    const { todos, onSelect, selectedUserId } = this.props;
 
-        <li className="TodoList__item TodoList__item--checked">
-          <label>
-            <input type="checkbox" checked readOnly />
-            <p>distinctio vitae autem nihil ut molestias quo</p>
-          </label>
+    const filteredTodos = todos
+      .filter(
+        ({ title }) => title.toLowerCase().includes(this.state.query),
+      ).filter(
+        ({ completed }) => {
+          switch (this.state.status) {
+            case 'active':
+              return !completed;
+            case 'completed':
+              return completed;
+            default:
+              return true;
+          }
+        },
+      );
 
-          <button
-            className="TodoList__user-button button"
-            type="button"
+    return (
+      <div className="TodoList">
+        <h2>Todos:</h2>
+        <label>
+          Search by title
+          <input
+            type="text"
+            onChange={event => this.handleChange(event, 'query')}
+          />
+        </label>
+        <select
+          onChange={event => this.handleChange(event, 'status')}
```

Comment:
get field's name from name attribute
```suggestion
               name='query'
               onChange={this.handleChange}
          />
        </label>
        <select
          name='status'
          onChange={this.handleChange}
```

Reply:
move input and select to App.js